website/content/en/preview/AWS/user-data.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "User Data Configuration"
+linkTitle: "UserData"
+weight: 10
+description: >
+  Learn how to configure custom UserData with Karpenter
+---
+
+This document describes how you can customize the UserData that will be specified on your EC2 worker nodes, without using a launch template.
+
+## Configuration
+
+In order to specify custom user data, you must include it within a AWSNodeTemplate resource. You can then reference this AWSNodeTemplate resource through `spec.providerRef` in your provisioner.
+
+**Examples**
+
+Your UserData can be added to `spec.userData` in the `AWSNodeTemplate` resource like this -
+```
```

Comment:
Can we specify in a comment that this is bottlerocket specific?

Reply:
Done